Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,117 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    git_URL=""
+    git_SHA=""
+    ruby_URL="https://github.com/xu-cheng/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.x86_64-darwin9.tar.gz"
+    ruby_SHA="b9e7a4e66dd70c33c0912d753f4ced6e3338b4eb6d348c250c6dd2fda4c809b5"
+  else
+    git_URL=""
+    git_SHA=""
+    ruby_URL=""
+    ruby_SHA=""
+  fi
+else
+  git_URL=""
+  git_SHA=""
+  ruby_URL=""
+  ruby_SHA=""
+fi
+
+cleanup-old() {
+  for version in "$HOMEBREW_LIBRARY/Homebrew/vendor/$VENDOR_NAME"/*
+  do
+    [[ -L "$version" ]] && continue # skip opt symlink
```

Comment:
:+1: for doc. 

> May just check the name too

You mean check both file type and opt name. Or only one of them?